modules/ROOT/partials/configuration/defaultmenuitems.adoc

```diff
@@ -3,13 +3,13 @@
 tinymce.init({
   selector: 'textarea',  // change this value according to your HTML
   menu: {
-    file: { title: 'File', items: 'newdocument restoredraft | preview | print ' },
-    edit: { title: 'Edit', items: 'undo redo | cut copy paste | selectall | searchreplace' },
-    view: { title: 'View', items: 'code | visualaid visualchars visualblocks | spellchecker | preview fullscreen' },
-    insert: { title: 'Insert', items: 'image link media template codesample inserttable | charmap emoticons hr | pagebreak nonbreaking anchor | insertdatetime' },
-    format: { title: 'Format', items: 'bold italic underline strikethrough superscript subscript codeformat | styles blocks fontfamily fontsize align lineheight | forecolor backcolor | removeformat' },
-    tools: { title: 'Tools', items: 'spellchecker spellcheckerlanguage | code wordcount' },
-    table: { title: 'Table', items: 'inserttable | cell row column | tableprops deletetable' },
+    file: { title: 'File', items: 'newdocument restoredraft | preview | export print | deleteallconversations' },
+    edit: { title: 'Edit', items: 'undo redo | cut copy paste pastetext | selectall | searchreplace' },
+    view: { title: 'View', items: 'code | visualaid visualchars visualblocks | spellchecker | preview fullscreen | showcomments' },
+    insert: { title: 'Insert', items: 'image link media addcomment pageembed template codesample inserttable | charmap emoticons hr | pagebreak nonbreaking anchor tableofcontents | insertdatetime' },
```

Comment:
Do we need a followup Jira to remove tableofcontents from the defaults?

Reply:
No because it is always in the defaults. It's what has to be done to get the items included in the menus, see: https://github.com/tinymce/tinymce/blob/236891ffcdf2fd1b698d020e67c31838fb110019/modules/tinymce/src/themes/silver/main/ts/ui/menus/menubar/Integration.ts#L25. You'll notice multiple other premium plugins are listed as well. To solve the part where they don't have to be listed in core is an entirely separate thing that should be discussed elsewhere if we want to go down that path.

Reply:
I should add if we just don't want premium plugins documented that's up to you 🤷